trackintel/io/file.py

```diff
@@ -74,14 +74,14 @@ def read_positionfixes_csv(*args, **kwargs):
     >>> trackintel.read_positionfixes_csv('data.csv')
     >>> trackintel.read_positionfixes_csv('data.csv', columns={'time':'tracked_at', 'User':'user_id'})
     """
-    columns = kwargs.pop('columns', {})
-    tz = kwargs.pop('tz', None)
-    crs = kwargs.pop('crs', None)
-    
+    columns = {} if columns is None else columns
+
     # Warning if no 'index_col' parameter is provided
-    if 'index_col' not in kwargs:
+    if index_col is None:
```

Comment:
I think here the behaviour is not what we expected, There are three cases:
1. User pass `index_col` explicitly, we pass it to pd.read_csv().
2. User do not pass `index_col`, we raise a warning.
3. User pass `index_col=None`, no warning should be raised as the user should be aware that we assume index as the unique identifier.

I think the third case is not covered here. I am not sure how to implement this though, is there a way to distinguish user passed None vs. the default None?